src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -21,9 +21,13 @@ We use Docker to replicate the production environment for the local deployment o
 
 Use the `appwrite run functions` command to develop your function locally.
 
+| parameter       | Description                                                                                                |
```

Comment:
Also `  --with-variables`, which sets your production env variables on function that you run locally. Very scary for security, only use it if your production function doesn't have production secrets in env variables, or you are very aware of what this can lead to.

Variables in `.env` of your functio are always loaded. this parameter does not affect it